README.md

```diff
@@ -55,6 +56,26 @@ Groups can be added manually or using the group.py script
 
 ![](CCV-auto-group.png)
 
+6. Acknowledging vulnerabilities in bulk
+
+6.1 --export-affected-devices: use ack_vuln.py with export-affected-devices options to export vulnerable devices into CVS file. This script takes two additional parameters \
```

Comment:
CSV (not CVS)

Also, should we say "This script takes.." or "This operation takes" ? (this comment applies to other operations below as well)